packages/core/src/components/widgets/AltDateTimeWidget.tsx

```diff
@@ -0,0 +1,18 @@
+import { WidgetProps } from "@rjsf/utils";
+import React from "react";
+import AltDateWidget from "./AltDateWidget";
+
+function AltDateTimeWidget<T = any, F = any>({
+  time = true,
+  ...props
+}: WidgetProps<T, F>) {
+  const { AltDateWidget } = props.registry.widgets;
+  return <AltDateWidget time={time} {...props} />;
+}
+
+AltDateTimeWidget.defaultProps = {
+  ...AltDateWidget.defaultProps,
+  time: true,
+};
+
```

Comment:
With the `time=true` above you can remove the `defaultProps`

Reply:
This was a bit more complex than expected. Looks like defaultProps actually merges objects as well (see new implementation).